src/RandMsg.chpl

```diff
@@ -30,14 +30,12 @@ module RandMsg
         var (lenStr,dtypeStr,aMinStr,aMaxStr,seed) = payload.splitMsgToTuple(5);
         var len = lenStr:int;
         var dtype = str2dtype(dtypeStr);
+        writeln(dtype);
```

Comment:
debugging?

Reply:
Poor Ben's going to be shamed into never opening another draft PR again.  :D  I'd hold off on reviewing details like this until he gets to the point of considering this no longer to be in draft form.  If he develops like me (throw in a bunch of code that I'll eventually remove), he'll look at the diff and boil it down to the essentials between now and then.

Reply:
I don't think he should feel ashamed this is a great help to us.
I learn stuff from his code all the time and I don't have time to be ashamed of anything.
Sorry if I am too hard. I wanted to see the code and thought I would make a few suggestions to get this off the ground ASAP.

@bmcdonald3 I think you and your code are awesome and I don't butter people up ;-)

Reply:
Thanks Mike and I definitely appreciate the feedback, I am far from expert in the client-side code especially.